modules/op-release-notes-1-13.adoc

```diff
@@ -140,3 +140,21 @@ include::snippets/technology-preview.adoc[]
 * Before this update, when using {tekton-chains}, if you configured the `storage.oci.repository` parameter, errors were reported in the Tekton Chains controller log. With this update, the `storage.oci.repository` parameter is processed correctly.
 
 * Before this update, when {tekton-chains} was configured with the Hashicorp Vault KMS and there was an issue with the connection to Vault, the {tekton-chains} controller pod could crash. With this update, the errors are processed and logged on the {tekton-chains} controller log.
+
+[id="release-notes-1-13-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.13.1
+
+With this update, {pipelines-title} General Availability (GA) 1.13.1 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-13-1_{context}"]
+=== Fixed issues
+
+* Before this update, a task run sometimes failed with a `cannot stop the sidecar` error message. With this update, the race condition between controllers that caused this failure is fixed.
+
+* Before this update, to enable keeping pods when a task run is cancelled, along with setting the `pipeline.options.configMaps.feature-flags.data.keep-pod-on-cancel` spec to `true` in the `TektonConfig` CR, you also need to set the `pipeline.enable-api-fields` spec to `alpha` in the `TektonConfig` CR. WIth this update, setting the `pipeline.options.configMaps.feature-flags.data.keep-pod-on-cancel` spec to `true` in the `TektonConfig` CR enables keeping pods when a task run is cancelled, and no additional setting is necessary.
+
+* With this update, if a sidecar is defined for a pipeline run and a webhook event triggers the pipeline run, {pipelines-shortname} validates the sidecar.
```

Comment:
Ah yeah the release-note upstream for this is bad. In the pull-request description there is "This commit fixes the inconsistent validations for Sidecars by adding webhook validations. It previously only validate Sidecar at runtime instead of what's validated for Step at webhook."

So this is about the container image used in the sidecar definition. Previously it wasn't validated at creation time but at runtime (so if it was invalid, previously it would fail at runtime, whereas now, it fails at creation time, and thus doesn't accept the creation)